packages/blockchain-link/src/workers/blockbook/index.ts

```diff
@@ -256,6 +352,14 @@ const subscribeBlock = async () => {
     return socket.subscribeBlock();
 };
 
+const subscribeFiatRates = async (currency?: string) => {
+    if (common.getSubscription('fiatRates')) return { subscribed: true };
+    const socket = await connect();
+    common.addSubscription('fiatRates');
+    socket.on('fiatRates', onNewFiatRates);
```

Comment:
allow multiple currencies?

Reply:
as we discussed, there will be only one active subscription at a time. is removing this line
```if (common.getSubscription('fiatRates')) return { subscribed: true };``` enough are there is more to it?